dcicutils/ff_utils.py

```diff
@@ -20,11 +20,13 @@
     from urllib.parse import urlencode
 
 
+# TODO (C4-92, C4-102): Probably to centralize this information in env_utils. Also figure out relation to CGAP.
 HIGLASS_BUCKETS = ['elasticbeanstalk-fourfront-webprod-wfoutput',
                    'elasticbeanstalk-fourfront-webdev-wfoutput']
 
 
 # TODO (C4-92): Centralize this information, it is repeated in other repos
+# TODO (C4-102): Does this need to include CGAP envs? As part of the same list, or as a separate list?
```

Comment:
It may make sense to add cgap as this package can be used on CGAP as well as Fourfront despite its name, though that may lead to some interesting complications.